### PR TITLE
Add new code Ruby

### PR DESCRIPTION
**Pull request by nguyenduyhao1111**

### CODE REVIEW

File: week0001/fizz_buzz_haond1.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FIZZ = [1,2,2]
+def fizz_buzz(n)
+  (n/5).times{|i|
+    FIZZ[i%3].times{ p 'fizz' }
+    p 'buzz'
```
Comment (vietlq) [inline week0001/fizz_buzz_haond1.rb]: Note that you must print `Fizz` & `Buzz`! Your current output gives double quotes on every single line. The correct output is in `correct_output.dat` (merge from my repo to yours).

Comment (nguyenduyhao1111) [inline week0001/fizz_buzz_haond1.rb]: Thanks ! :+1:

**Comment by vietlq** [comment]
Thanks for the quick update. I can see that you are almost there:

```
$ wc <(ruby fizz_buzz_haond1.rb)
     53      53     371 /dev/fd/63
```

Can you please remove the quotes?
```
# Sample run
ruby fizz_buzz_haond1.rb
"Fizz"
"Buzz"
"Fizz"
"Fizz"
...
```

Thanks.

**Comment by nguyenduyhao1111** [comment]
Please run 
```
ruby fizz_buzz_haond1.rb
```
Then output :
```
Fizz
Buzz
Fizz
Fizz
Buzz
Fizz
Fizz
...
```

**Pull request merged**